scormxblock/scormxblock.py

```diff
@@ -66,10 +66,8 @@ def updoad_all_content(temp_directory, fs):
     This standalone function handles the bulk upload of unzipped content.
     """
     if not settings.DJFS.get('type', 'osfs') == "s3fs":
-        # Temporary fix
-        # TODO: find a better solution for ImportError: No module named fs.utils
-        from fs.utils import copydir
-        copydir(temp_directory, fs, overwrite=True)
+        from fs.copy import copy_fs
+        copy_fs(temp_directory, fs)
```

Comment:
it's not necessary to specify the overwrite parameter ?

Reply:
nope, in fact, copy_fs does not have that parameter https://docs.pyfilesystem.org/en/latest/reference/copy.html#fs.copy.copy_fs